catalogue/webapp/components/RequestLocation/RequestLocation.js

```diff
@@ -314,8 +318,10 @@ const RequestLocation = ({ work }: Props) => {
               {!showRequestModal && (
                 <div data-test-id="resultsModalCTA">
                   <Modal
+                    id="resultsModal"
                     isActive={showResultsModal}
                     setIsActive={setShowResultsModal}
+                    openButtonRef={{ current: null }}
```

Comment:
Assuming this is just making the type system happy?

Reply:
yep, didn't dig into it deeper as it can be tackled as part of the requesting work